include/magma/vk/buffers/Buffer.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <string>
+#include <vulkan/vulkan.h>
+#include <vulkan/vulkan_core.h>
+
+struct BufferInfo {
+    VkDevice                device;
+    VkBufferCreateInfo      bufferInfo;
+    VkMemoryPropertyFlags   memoryProperty;
+    std::string             name;
+};
+
+class Buffer {
+friend class BufferManager;
+
+public:
+    Buffer() = default;
+    const VkBuffer          &getBuffer()    const { return _buffer;         }
+    const VkDeviceMemory    &getMemory()    const { return _bufferMemory;   }
```

Comment:
I'd prefer shorter version of getters, like:
```
getBuf()
getMem()
```

Reply:
Done